int64 can_id
---
bool result